packages/mint-components/src/components/sqm-share-button/sqm-share-button.tsx

```diff
@@ -56,6 +56,18 @@ export class ShareButton {
   //  Optional styling attrs
   //
 
+  /**
+   * @uiName Configure border radius with pixel amount
+   */
+  @Prop() borderradius?: number;
```

Comment:
This applies in other places, but I think camel case is the standard we want to follow and should enforce for new props being added

Reply:
Done! Changed all other props to camel case in share button